server/php/UploadHandler.php

```diff
@@ -896,32 +903,33 @@ protected function imagick_create_scaled_image($file_name, $version, $options) {
         $image_oriented = false;
         if (!empty($options['auto_orient'])) {
             $image_oriented = $this->imagick_orient_image($image);
-        } 
-	    
-        $image_resize = false; 
+        }
+
+        $image_resize = false;
         $new_width = $max_width = $img_width = $image->getImageWidth();
-        $new_height = $max_height = $img_height = $image->getImageHeight(); 
-		  
+        $new_height = $max_height = $img_height = $image->getImageHeight();
+        $x = $y = 0;
```

Comment:
This should not be necessary, as in all cases where `$x` or `$y` are used, they're initialized first.

Reply:
You're correct, sorry. My IDE went wild here I guess, showing me an issue where there actually isn't one. I didn't catch that.

Reply:
No problem, but I'd definitely prefer to only add code lines that are necessary.

I also think the newlines don't really add much here and the convention for this source code file is to keep code inside of functions without extra newlines.